src/extract.js

```diff
@@ -50,11 +50,23 @@ exports.TranslationReference = class TranslationReference {
 };
 
 function preprocessScriptTags(data, type) {
+  if (type !== 'vue') {
+    return {};
+  }
+
   const vueFile = vueCompiler.parse({ source: data, needMap: false });
+  if (!vueFile.script) {
+    return {};
+  }
 
-  return (type !== 'vue' || ! vueFile.script)
-    ? ''
-    : vueFile.script.content.trim();
+  const script = {};
+  if (vueFile.script.content) {
+    script.content = vueFile.script.content.trim();
+  }
+  if (vueFile.script.lang) {
+    script.lang = vueFile.script.lang.trim();
+  }
+  return script;
```

Comment:
I think that if there is a ``vueFile.script`` you will have the lang and the content (or it's weird !)

So just return like
```javascript
return {
  script: vueFile.script.content.trim(),
  lang: vueFile.script.lang,
}
```
why would you trim() the lang ? :)

Reply:
> So just return like

Looks better! I changed the code at 262d4e4.

> why would you trim() the lang ? :)

I added it just in case.
It's no need so I removed the trim() for the lang at also 262d4e4.